src/extension.ts

```diff
@@ -129,6 +132,8 @@ export async function activate(context) : Promise<extensionapi.ExtensionAPI> {
         registerCommand('extension.vsKubernetesDashboard', dashboardKubernetes),
         registerCommand('extension.vsKubernetesCopy', copyKubernetes),
         registerCommand('extension.vsKubernetesPortForward', portForwardKubernetes),
+        registerCommand('extension.vsKubernetesLoadConfigMapData', configmaps.loadConfigMapData),
+        registerCommand('extension.vsKubernetesDeleteFile', (obj) => { deleteKubernetesConfigFile(obj, treeProvider) }),
```

Comment:
tslint: needs semicolon in the body of the lambda

Reply:
done.